src/transformStateWithClones.js

```diff
@@ -8,6 +8,40 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const statefulClone = {
```

Comment:
`statefulCloneCloned` are not very descriptive. Consider using names that describe what the variables represent, in this case it's hard to guess. 🤔 Maybe try sth like statefulCloneHistory